From f27c0f727a8ee48591996178efccbd235d651574 Mon Sep 17 00:00:00 2001
From: Dirk Mueller <dirk@dmllr.de>
Date: Wed, 7 Mar 2018 23:00:23 +0100
Subject: [PATCH] Updating time for functional tests

Currenlty, some gates is being failed with logs like this

"""
b'Response: {"description": "Provided object does not match schema
\'Secret\': \'expiration\' is before current time.
Invalid property: \'expiration\'", "title": "Bad Request", "code": 400}'
"""

So it is necessary to update the time to create secrets successfully.

Change-Id: I4f2d3817d9a2380919d16e76ef5dcba3ec62e5a1
---
 functionaltests/client/v1/functional/test_acl.py        | 2 +-
 functionaltests/client/v1/functional/test_containers.py | 2 +-
 functionaltests/client/v1/functional/test_secrets.py    | 2 +-
 functionaltests/client/v1/smoke/test_containers.py      | 2 +-
 functionaltests/client/v1/smoke/test_secrets.py         | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/functionaltests/client/v1/functional/test_acl.py b/functionaltests/client/v1/functional/test_acl.py
index 61de646..d9f8fc8 100644
--- a/functionaltests/client/v1/functional/test_acl.py
+++ b/functionaltests/client/v1/functional/test_acl.py
@@ -24,7 +24,7 @@ from barbicanclient import exceptions
 
 create_secret_defaults_data = {
     "name": "AES key",
-    "expiration": "2018-02-28T19:14:44.180394",
+    "expiration": "2020-02-28T19:14:44.180394",
     "algorithm": "aes",
     "bit_length": 256,
     "mode": "cbc",
diff --git a/functionaltests/client/v1/functional/test_containers.py b/functionaltests/client/v1/functional/test_containers.py
index e632b23..64b9cdc 100644
--- a/functionaltests/client/v1/functional/test_containers.py
+++ b/functionaltests/client/v1/functional/test_containers.py
@@ -22,7 +22,7 @@ from barbicanclient import exceptions
 
 create_secret_defaults_data = {
     "name": "AES key",
-    "expiration": "2018-02-28T19:14:44.180394",
+    "expiration": "2020-02-28T19:14:44.180394",
     "algorithm": "aes",
     "bit_length": 256,
     "mode": "cbc",
diff --git a/functionaltests/client/v1/functional/test_secrets.py b/functionaltests/client/v1/functional/test_secrets.py
index 7832154..8d9fcf0 100644
--- a/functionaltests/client/v1/functional/test_secrets.py
+++ b/functionaltests/client/v1/functional/test_secrets.py
@@ -28,7 +28,7 @@ from testtools import testcase
 
 secret_create_defaults_data = {
     "name": "AES key",
-    "expiration": "2018-02-28T19:14:44.180394",
+    "expiration": "2020-02-28T19:14:44.180394",
     "algorithm": "aes",
     "bit_length": 256,
     "mode": "cbc",
diff --git a/functionaltests/client/v1/smoke/test_containers.py b/functionaltests/client/v1/smoke/test_containers.py
index 2c67bca..584df47 100644
--- a/functionaltests/client/v1/smoke/test_containers.py
+++ b/functionaltests/client/v1/smoke/test_containers.py
@@ -20,7 +20,7 @@ from testtools import testcase
 
 create_secret_defaults_data = {
     "name": "AES key",
-    "expiration": "2018-02-28T19:14:44.180394",
+    "expiration": "2020-02-28T19:14:44.180394",
     "algorithm": "aes",
     "bit_length": 256,
     "mode": "cbc",
diff --git a/functionaltests/client/v1/smoke/test_secrets.py b/functionaltests/client/v1/smoke/test_secrets.py
index 5482582..3f11740 100644
--- a/functionaltests/client/v1/smoke/test_secrets.py
+++ b/functionaltests/client/v1/smoke/test_secrets.py
@@ -21,7 +21,7 @@ from testtools import testcase
 
 secret_create_defaults_data = {
     "name": "AES key",
-    "expiration": "2018-02-28T19:14:44.180394",
+    "expiration": "2020-02-28T19:14:44.180394",
     "algorithm": "aes",
     "bit_length": 256,
     "mode": "cbc",
-- 
2.16.2

